Installing on Ubuntu 16.04

During the making of the install is shown this error:

``` bash

make
make  all-recursive
make[1]: ingresso nella directory "/home/kappa95/Documenti/libwacom"
Making all in libwacom
make[2]: ingresso nella directory "/home/kappa95/Documenti/libwacom/libwacom"
  CC       libwacom_la-libwacom.lo
libwacom.c: In function ‘libwacom_copy’:
libwacom.c:321:15: warning: implicit declaration of function ‘g_array_copy’ [-Wimplicit-function-declaration]
  d->matches = g_array_copy(device->matches);
               ^
libwacom.c:321:13: warning: assignment makes pointer from integer without a cast [-Wint-conversion]
  d->matches = g_array_copy(device->matches);
             ^
  CC       libwacom_la-libwacom-error.lo
  CC       libwacom_la-libwacom-database.lo
  CC       libwacom_la-libwacom-deprecated.lo
  CCLD     libwacom.la
ar: `u' modifier ignored since `D' is the default (see `U')
make[2]: uscita dalla directory "/home/kappa95/Documenti/libwacom/libwacom"
Making all in data
make[2]: ingresso nella directory "/home/kappa95/Documenti/libwacom/data"
Making all in layouts
make[3]: ingresso nella directory "/home/kappa95/Documenti/libwacom/data/layouts"
make[3]: Nessuna operazione da eseguire per "all".
make[3]: uscita dalla directory "/home/kappa95/Documenti/libwacom/data/layouts"
make[3]: ingresso nella directory "/home/kappa95/Documenti/libwacom/data"
make[3]: Nessuna operazione da eseguire per "all-am".
make[3]: uscita dalla directory "/home/kappa95/Documenti/libwacom/data"
make[2]: uscita dalla directory "/home/kappa95/Documenti/libwacom/data"
Making all in doc
make[2]: ingresso nella directory "/home/kappa95/Documenti/libwacom/doc"
make[2]: Nessuna operazione da eseguire per "all".
make[2]: uscita dalla directory "/home/kappa95/Documenti/libwacom/doc"
Making all in test
make[2]: ingresso nella directory "/home/kappa95/Documenti/libwacom/test"
  CC       test_load-test-load.o
  CCLD     test-load
../libwacom/.libs/libwacom.so: undefined reference to `g_array_copy'
collect2: error: ld returned 1 exit status
Makefile:667: set di istruzioni per l'obiettivo "test-load" non riuscito
make[2]: *** [test-load] Errore 1
make[2]: uscita dalla directory "/home/kappa95/Documenti/libwacom/test"
Makefile:468: set di istruzioni per l'obiettivo "all-recursive" non riuscito
make[1]: *** [all-recursive] Errore 1
make[1]: uscita dalla directory "/home/kappa95/Documenti/libwacom"
Makefile:377: set di istruzioni per l'obiettivo "all" non riuscito
make: *** [all] Errore 2
```

--- a/libwacom/libwacom.c
+++ b/libwacom/libwacom.c
@@ -318,9 +318,13 @@ libwacom_copy(const WacomDevice *device)
 	d->height = device->height;
 	d->integration_flags = device->integration_flags;
 	d->layout = g_strdup(device->layout);
-	d->matches = g_array_copy(device->matches);
-	for (guint i = 0; i < d->matches->len; i++)
-		libwacom_match_ref(g_array_index(device->matches, WacomMatch*, i));
+	d->matches = g_array_sized_new(TRUE, TRUE, sizeof(WacomDevice*),
+				       device->matches->len);
+	for (guint i = 0; i < device->matches->len; i++) {
+		WacomMatch *m = g_array_index(device->matches, WacomMatch*, i);
+		libwacom_match_ref(m);
+		g_array_append_val(d->matches, m);
+	}
 	d->match = device->match;
 	if (device->paired)
 		d->paired = libwacom_match_ref(device->paired);

